### DISCUSSION TITLE
Two ACs API V1 and V2

### DISCUSSION THREAD

**Discussion by WojtekWro** [nVuln/homebridge-lg-thinq#189] [General]
Hi, 
I  just bought another LGAC. The first one was working with this plugin until I installed the new one. the first one uses api v1, the new one uses api V2. 
It is possible to those both ACs work with this plugin?

**Comment by nVuln** [comment]
yup, v2 working fine with this plugin
v1 still work but you need enable v1 support in plugin setting first

**Comment by WojtekWro** [comment]
I confirm. It works. 
However I am wondering how it works. I mean I have two ACs visible in one application from LG, how LG manage this in their infrastructure? Is there a possibility to upgrade firmware in my AC which supports API v1?

**Reply by nVuln** [reply]
as my knowledge, you cannot upgrade firmware from v1 to v2, it's fixed in wifi module chip (change wifi module may possible, I'm not sure about this 👻)
in LG app, they have single api server to list all devices (response all linked device include device's thinq version)
based on version they have different way to monitor & control it
